### ISSUE TITLE
Change event stopped working well

### ISSUE THREAD

**Issue by magoarcano** [ngx-formly/ngx-formly#2390]
**Description**
For input type fields, in the `templateOptions.change` callback, the parameter `event` is not working anymore. It has the same value as `field` parameter.

**Minimal Reproduction**
```
{
                  key: 'code',
                  type: 'input',
                  templateOptions: {
                      label: 'Procedure code',
                      change: (field, event) => {
                        console.log(field === event); // prints true
                        this.doSomething(field, event.target.value); // TypeError: Cannot read property 'value' of undefined
                      },
                  },
```

In the next stackblitz, if you change the value of the first input, it shows in console that `field === event` is `true` and `event.target` is `undefined`
https://stackblitz.com/edit/ngx-formly-ui-material-j8rukg

**Your Environment**

- Angular version: 9.1.12
- Formly version: 5.9.1

**Additional context**
I think the problem started between versions 5.7.2 and 5.8.0 of Formly.

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.9.2](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.9.2) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!

**Comment by magoarcano** [comment]
Thank you very much for the fast response and fix!

That issue with input type field is fixed, but it seems it's not working for custom fields.

I have a custom field with the following template:
```
<ng-select class="custom"
    [items]="to.options | async" [bindValue]="to.valueProp" [bindLabel]="to.labelProp"
    [searchFn]="to.searchFn" [addTag]="to.addTag" [addTagText]="to.addTagText"
    [clearable]="to.clearable" [notFoundText]="to.notFoundText"
    [placeholder]="to.placeholder" [formControl]="formControl" [formlyAttributes]="field">
</ng-select>
```

```
templateOptions: {
        label: 'Ng select',
        change: (field, event) => console.log(event),
      }
```
When I change the value, the event is not running at all. It was working before with formly version 5.7.2

**Comment by aitboudad** [comment]
fixed in [v5.9.3](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.9.3) release!
